### DISCUSSION TITLE
No mastodon icon on my profile

### DISCUSSION THREAD

**Discussion by raspbeguy** [Traewelling/traewelling#2534] [Questions / Support]
Hello, I'm wondering why there is no mastodon icon on [my profile page.](https://traewelling.de/@raspbeguy)

I linked my fedi account (a GoToSocial instance), actually I create my Träwelling account by logging in from fedi authentication. It shows that I'm correctly connected in the "login providers" page.

Also the "link ownership" feature on mastodon doesn't seem to work for me. Because my Träwelling profile doesn't seem to acknowledge my fedi account, the link I put in my mastodon bio doesn't show as "owned".

**Comment by MrKrisKrisu** [comment]
Hi, it seems that this is a bug in Träwelling. I don't see your account connected in the admin panel either. But when I look in the database I see that you have a server assigned. I'll have a look in the code to see what the problem might be.

![image](https://github.com/Traewelling/traewelling/assets/4103693/a5d6ebec-9f52-46d8-b2c6-e0f16e622551)

**Reply by MrKrisKrisu** [reply]
I have just seen that the instance you are connecting to is [GoToSocial](https://github.com/superseriousbusiness/gotosocial). Träwelling does not currently fully support this - we use an API from Mastodon to verify the account, which is not available on your instance. Therefore, your account is unknown and will not be displayed.

```
Client error: `GET https://{url}/api/v1/accounts/{id}` resulted in a `404 Not Found` response
```

**Reply by raspbeguy** [reply]
This call is supported by gotosocial (and my instance). I just tested it and it works.

Are you trying to use the username instead of the user ID ?
